pkg/controller/apiserver/apiserver_controller.go

```diff
@@ -268,7 +268,7 @@ func (r *ReconcileAPIServer) Reconcile(request reconcile.Request) (reconcile.Res
 	}
 
 	// Everything is available - update the CRD status.
-	instance.Status.State = operatorv1.APIServerStatusReady
+	instance.Status.State = operatorv1.TigeraStatusReady
```

Comment:
I think this is fine, but for context we intentionally had different definitions for the different CR status fields in case we needed to represent different statuses for different CRs.

I suppose we can always split again if the need arises.

Reply:
This was my suggestion, I haven't seen a case where we've actually done that and @rene-dekker was following the pattern and adding another "Ready" field. I figured that if something wants a different status it could define it's own and use it, but if it doesn't need it there's a generic one.